lib/stat_tracker.rb

```diff
@@ -0,0 +1,16 @@
+require "csv"
+require_relative './game'
+
+class StatTracker
+  attr_reader :games
+
+  def self.from_csv(files)
+    StatTracker.new(files)
+  end
+
+  def initialize(files)
+    @games = (CSV.open files[:games], headers: true, header_converters: :symbol).map do |row|
+      game = Game.new(row)
```

Comment:
Will need to add @teams and @game_teams classes to add in the StatTracker initialize next